plugins/inputs/couchbase/couchbase.go

```diff
@@ -36,10 +40,17 @@ var sampleConfig = `
 
   ## Filter bucket fields to include only here.
   # bucket_stats_included = ["quota_percent_used", "ops_per_sec", "disk_fetches", "item_count", "disk_used", "data_used", "mem_used"]
+
+  ## Optional TLS Config
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+  ## Use TLS but skip chain & host verification (defaults to true)
```

Comment:
With this latest change, insecure_skip_verify defaults to false, right?
```suggestion
  ## Use TLS but skip chain & host verification (defaults to false)
```

Reply:
Yes, good catch